ds3-autogen-net/src/main/java/com/spectralogic/ds3autogen/net/generators/elementparsers/BaseNullableElement.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * ******************************************************************************
+ *   Copyright 2014-2015 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+package com.spectralogic.ds3autogen.net.generators.elementparsers;
+
+import static com.spectralogic.ds3autogen.utils.Helper.uncapFirst;
+
+/**
+ * Represents an element within a .net model parser whose data
+ * is within an xml tag
+ */
+public class BaseNullableElement implements NullableElement {
+
+    protected final String name;
```

Comment:
If i add getters, then yes. Other types extend this one, so I need access, but a getter would do that cleaner